operator/v1/types_dns.go

```diff
@@ -48,6 +48,16 @@ type DNSSpec struct {
 	// +optional
 	Servers []Server `json:"servers,omitempty"`
 
+	// upstreamResolvers defines a schema for configuring CoreDNS
+	// to proxy DNS messages to upstream resolvers for the case of the
+	// default (".") server
+	//
+	// If this field is not specified, the upstream used will default to
+	// /etc/resolv.conf, with policy "sequential"
+	//
+	// +optional
+	UpstreamResolvers UpstreamResolvers `json:"upstreamResolvers"`
```

Comment:
this lacks a default value, probably `{}` via the yaml patch (kubebuilder is broken).